afl-persistent-config

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+# written by jhertz
+# 
+
+PLATFORM=`uname -s`
+
+
+# check that we're on Mac
+if [[ ! "$PLATFORM" = "Darwin" ]] ; then
+   echo "This script is for Mac OS"
+   exit 1
+fi
+
+# check if UID == 0
+if [[ $EUID -ne 0 ]]; then
+   echo "You really should be root to do this"
+   exit 1
+fi
+
+# check if SIP is disabled
+if [[ ! $(csrutil status | grep "disabled") ]]; then
+   echo "SIP needs to be disabled"
```

Comment:
One line on how to disable SIP here, else they'll ask..

Reply:
already done if you look at the current state :)